lib/opentok.js

```diff
@@ -901,6 +901,52 @@ OpenTok = function (apiKey, apiSecret, env) {
     });
   };
 
+  /**
+   * Send DTMF digits to all participants in an active OpenTok session or to a specific client
+   * connected to that session.
+   *
+   * @param sessionId The session ID corresponding to the session that will receive the DTMF string.
+   *
+   * @param connectionId The connection connection ID of the client you are sending the DTMF signal to.
+   *  Leave this empty to send a DTMF signal to all clients connected to the session.
+   *
+   * @param digits This is the string of DTMF digits to send. This can include 0-9, '*', '#', and
+   * 'p'. A p indicates a pause of 500ms (if you need to add a delay in sending the digits).
+   *
+   * @param callback {Function} The function to call upon completing the operation. One argument
+   * is passed to the function:
+   *
+   * <ul>
+   *
+   *   <li>
+   *      <code>error</code> &mdash; An error object (if the call to the method fails). This is
+   *      set to null if there is no error.
+   *   </li>
+   *
+   * </ul>
+   * @return
```

Comment:
Please delete this line. (It causes an error when running jsdoc (because of the empty return declaration).

Reply:
@hamzanasir interesting the tests failed for Jeff but not in the CI.  Any ideas?